src/coreclr/src/jit/emitxarch.cpp

```diff
@@ -14818,6 +14819,8 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             result.insLatency += PERFSCORE_LATENCY_2C;
             break;
 
+        case INS_bsf:
+        case INS_bsr:
```

Comment:
It would be nice if we had a way to model Intel vs AMD here.

| Instruction | Intel | AMD  |
| ----------- | ----- | ---- |
| BSF         | 1.00  | 4.00 |
| BSR         | 1.00  | 5.00 |
| POPCNT      | 1.00  | 0.50 |
| LZCNT       | 1.00  | 0.50 |
| TZCNT       | 1.00  | 0.50 |

* For the non-memory encoding, AMD is actually a bit faster still
* Numbers taken from https://uops.info/table.html